rust/Cargo.lock

```diff
@@ -7037,6 +7037,7 @@ dependencies = [
  "num-traits",
  "once_cell",
  "prometheus",
+ "rand 0.5.6",
```

Comment:
why is the version here different to the one in the Cargo.toml?

Reply:
![Screen Shot 2024-06-24 at 2 24 42 PM](https://github.com/hyperlane-xyz/hyperlane-monorepo/assets/20362969/85de5964-94b1-4fff-898b-e4895230ab04)

Whoa i made a typo! and we have this same typo in a bunch of other places that I've just fixed. Wild!

It shows up as a compilation warning but we have a lot of noise from some Solana warnings that it was ignored

Reply:
nice catch